src/clojure_ttt/ui.clj

```diff
@@ -55,17 +55,42 @@
        (some (partial = move))
        (boolean)))
 
-(defn input-move [board]
-  (let [s (input)]
-    (cond
+(defn input-move
+  ([board]
+   (input-move board 0))
+
+  ([board depth]
+   (cond
+     (= depth 0) (output "Please choose a move")
+     :else (output "Please choose a valid move"))
+
+   (let [s (input)]
+     (cond
+
+       (parseable-as-int? s)
```

Comment:
Can we combine the two conditions i.e. (and parseable-as-int? move-available) to save some duplication?